### ISSUE TITLE
More global preferences, please

### ISSUE THREAD

**Issue by hlship** [cursive-ide/cursive#1757]
Every time I create a new project, I have to go through some standard steps:

- Drag the REPL window to the corner I prefer
- Update my default REPL launch parameters the way I like them (e.g., turn off build, turn off tool window, add +pretty to profile, etc.)

Could some of these be derived from global preferences, rather than re-created every project?

kthanxbye

**Comment by cursive-ide** [comment]
So the REPL window corner I thought was persistent, but perhaps it's not. The others can be configured by using *Run&rarr;Edit configurations...* and looking under the Defaults tree - these are sort of like templates that new configs are created from. Editing the REPL config there will set the defaults for all new configs from that template.

**Comment by hlship** [comment]
This is true on an existing project, my concern is that when I create a *new* project I got through all this again. And again. And again.

**Comment by cursive-ide** [comment]
I see, yes - those defaults are stored in `workspace.xml`, I thought they were global.
